components/governance/org.wso2.carbon.governance.api/src/main/java/org/wso2/carbon/governance/api/util/GovernanceUtils.java

```diff
@@ -407,6 +407,12 @@ public static GovernanceArtifactConfiguration getGovernanceArtifactConfiguration
                     configuration.setLifecycle(lifecycleElement.getText());
                 }
 
+                OMElement taxonomyElement = configElement.getFirstChildWithName(
+                        new QName("taxonomies"));
```

Comment:
Thank you Niranjan for mentioning it,
This fixed with https://github.com/wso2/carbon-governance/pull/255